mypyc/primitives/generic_ops.py

```diff
@@ -281,7 +281,7 @@
         object_rprimitive,
     ],  # Keyword arg names tuple (or NULL)
     return_type=object_rprimitive,
-    c_function_name="_PyObject_Vectorcall",
+    c_function_name="PyObject_Vectorcall",
```

Comment:
`PyObject_Vectorcall` is available unconditionally from `pythoncapi_compat.h`. Included the underscore removal here so I wouldn't have to do another big PR just for that.

https://github.com/python/mypy/blob/aa0b6f0288e6a511b750f7fe8f49a0e321362105/mypyc/lib-rt/pythoncapi_compat.h#L602-L610